app/utils/press-release-snapshot.js

```diff
@@ -0,0 +1,56 @@
+import { tracked } from '@glimmer/tracking';
+
+/**
+ * Snapshot of a PressRelease (Pers bericht) record and related records to keep track of changes,
+ * because ember-data lacks dirty tracking for relationships and attributes of type 'array'.
+ *
+ * Contains application-specific logic to track the dirty state of relationships.
+ * If the model of a Source changes in the future, this class will probably require an update as well.
```

Comment:
Kan je de comments in deze file aanpassen van `source` naar `press-release` (ook voor de andere methods in deze class)